_posts/2020-08-20-codestarts-preview.adoc

```diff
@@ -0,0 +1,114 @@
+
+This preview version is just the beginning and includes only 3 example codestarts:
+
+- The legendary RestEasy example in Java, Kotlin and Scala, when you pick RestEasy extension
+- The lovely and sweet template engine Qute in Java, when you pick Qute extension
+- The tough and strong Quarkus Command Mode in Java, when you don’t pick any extension that provides an example
+
+== Wanna give it a try?
+
+We added a feature flag in the Quarkus Maven plugin to create your application:
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:create -DcodestartsEnabled`
+----
+
+=== A Command mode Quarkus project using Maven as build tool:
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:create -DcodestartsEnabled \
+-DprojectGroupId=org.acme -DprojectArtifactId=go-go-commando
+cd go-go-commando
+# ./mvnw.cmd on Windows
+./mvnw package
+java -jar target/go-go-commando-1.0-SNAPSHOT-runner.jar
+----
+
+===  Same as previous using Gradle as build tool:
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:create -DcodestartsEnabled \
+-DprojectGroupId=org.acme -DprojectArtifactId=go-go-commando-gradle \
+-DbuildTool=GRADLE
+cd go-go-commando-gradle
+# ./gradlew.bat on Window
+./gradlew build
+java -jar build/go-go-commando-gradle-1.0-SNAPSHOT-runner.jar
+----
+
+=== A Quarkus project in Kotlin with example for RESTEasy and Qute:
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:create -DcodestartsEnabled \
+-DprojectGroupId=org.acme -DprojectArtifactId=kotlin-with-rest-is-qute \
+-Dextensions=kotlin,resteasy,qute
+cd kotlin-with-rest-is-qute
+# ./mvnw.cmd on Windows
+./mvnw quarkus:dev
+----
+
+=== A blank Quarkus project in Scala with SmallRye Reactive Messaging Kafka
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:create -DcodestartsEnabled \
+-DprojectGroupId=org.acme -DprojectArtifactId=scala-kafka \
+-Dextensions=scala,quarkus-smallrye-reactive-messaging-kafka \
+-DskipExample
+cd scala-kafka
+# ./mvnw.cmd on Windows
+./mvnw quarkus:dev
+----
+
+NOTE: We added instruction to get you started (contextual to the extensions and build tool you've selected) in the new project `README.md` .
+
+== Start a blank Quarkus application with your favorites tools/extensions
+
+Codestarts also introduce a huge improvement 😅 and don’t laugh, it has been a legitimate demand from the community for a while now,
+using codestarts, you can generate a blank Quarkus app by adding: `-DskipExample`.
+
+== You might notice a few missing features with Codestarts
+
+1. The package name of the generated examples is now fixed to `org.acme`,
+we thought it wouldn’t add much value to make it customizable, but let us know if you find it’s a loss.
+2. Since a project will contain multiple examples, we must avoid conflicts, this is why you can't define the REST `path` or your app anymore.
```

Comment:
```suggestion
2. Since a project will contain multiple examples, we must avoid conflicts, this is why you can't define the REST `path` in your app anymore.
```
not sure if 'in' or 'of' or you actually meant some other settings besides path ?